### ISSUE TITLE
Import and apply previous mapping in one step

### ISSUE THREAD

**Issue by PRijnbeek** [OHDSI/Usagi#106]
Scenario:

I have a new version of the source code count and potentially also new source codes.

Expected behaviour

I can open the new one and apply the older mapping without waiting for importing the new one.
Only codes that are not in the old one are processed.

Could be a file button on the left where you can select that previous file

Current behaviour:

I import the new file, then wait a long time till all are processes, and then change all of them base on the old mapping.

**Comment by MaximMoinat** [comment]
@PRijnbeek I can see that this functionality would be helpful in some cases. I tried to reword the requirement below. Please let me know if I understood it correctly.

### Current process
1. Import updated code list. Automatic mapping suggestions are created by Usagi.
2. Apply previous mapping. Applying already created mappings and approvals. This probably overwrites a lot of the mappings made by Usagi made in step 1.
3. Review mapping suggestions for the new codes

### Suggested process
1. Open existing mapping
2. Import updated code list. For each source code:
    a. If code in opened mapping, update the code name and frequency (additional info?)
    b. If not, make an automatic mapping as normal and add to the opened mapping
3. Review mapping suggestions for the new codes (highlight the new codes?)

### Implementation
The easiest way to implement this would be with a check-box in the import menu, that is enabled if you had a mapping open already (e.g. _"[ ] Merge with opened mapping?"_ + explanation in hover-over).

**Comment by PRijnbeek** [comment]
Yes this sounds great.  I woud also add the Additional Info fields that are in the existing mapping. For example we add a field with the date of the first and last occurrence of the code in the data, values of measurements etc.
